CheckDrive.Web/CheckDrive.Web/Views/Shared/_ModalForLayout.cshtml

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
rename this view to **_LayoutForModals**

Reply:
fixed